frontend/dockerfile/builder/build.go

```diff
@@ -30,7 +30,6 @@ const (
 	DefaultLocalNameDockerfile = "dockerfile"
 	keyTarget                  = "target"
 	keyFilename                = "filename"
-	keyCacheFrom               = "cache-from"    // for registry only. deprecated in favor of keyCacheImports
```

Comment:
Hold this until Moby migrates to the new API (Migration should be fairy easy, opening Moby PR is appreciated :smiley: )
https://github.com/moby/moby/blob/0ef2e175e6441f3894ee6f517edb09908f79f62a/builder/builder-next/builder.go#L267

Reply:
@chendave do you want to work on this?

Reply:
@AkihiroSuda , Hi, sorry for the late response!

Cause I don't have many knowledge on moby itself, not sure I can work it out on time, it could be easier for someone else to fix it :)

Reply:
Older daemon should be still compatible with newer frontend, so legacy key needs to be supported

Reply:
I am not able to understand this comment, why should legacy key be supported for a newer frontend?  newer frontend shouldn't use the new key for build? or ask in this way, what's the problem with the dropping of legacy key here?

Due to my limited knowledge, I am going to revert the change back.